apps/question-bank/build.gradle

```diff
@@ -87,9 +87,13 @@ dependencies {
 
     // id generation
     implementation project(':id-generator')
+    
+    
+    implementation project(':database-entities')
 
 }
 
+compileJava.mustRunAfter(":database-entities:build")
```

Comment:
Since the only Entity that is being pulled in from the database-entities is the `CodeValueGeneral` entity, lets just copy it over.